lib/types/result-set.js

```diff
@@ -148,6 +148,8 @@ function getCqlObject(field) {
       return field.getCounter();
     case rust.CqlTypes.Double:
       return field.getDouble();
+    case rust.CqlTypes.Duration:
+      return field.getDuration();
     case rust.CqlTypes.Float:
       return field.getFloat();
```

Comment:
A fundamental question: why were other types already present here, and Duration was not? What makes Duration different in this context?

Reply:
Those types were added in branch ``types-support``